idd/Energy+.idd.in

```diff
@@ -76235,6 +76252,42 @@ FaultModel:HumidistatOffset,
        \note Required field for Type ThermostatOffsetDependent
        \type object-list
        \object-list ThermostatOffsetFaults   
+	   
+FaultModel:Fouling:AirFilter,
+   \memo This object describes fault of dirty air filters
+   \min-fields 4
+   A1, \field Name
+       \note Enter the name of the fault
+       \required-field
+       \type alpha
+   A2, \field Fan Name
+       \note Enter the name of a fan object
+       \required-field
+       \type object-list
+       \object-list FansCVandOnOffandVAV 
+   A3, \field Fan Object Type
```

Comment:
@rongpengzhang Fan Object Type field should be before Fan Object Name field to follow the pattern used elsewhere.